app/controllers/api/public/v1/providers/locations_controller.rb

```diff
@@ -0,0 +1,27 @@
+module API
+  module Public
+    module V1
+      module Providers
+        class LocationsController < API::Public::V1::ApplicationController
+          def index
+            render jsonapi: locations, class: API::Public::V1::SerializerService.execute
+          end
+
+        private
+
+          def locations
+            @locations ||= provider.courses.find_by(course_code: params[:course_code])&.sites
```

Comment:
implement `def course` too for obviousness?

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1513/commits/567611daa99b3a12658d1cd168f9723158714866